### PR TITLE
Remove Google+ in install page

### LINKED ISSUE #20267: Remove Google+ in install page
## Is your feature request related to a problem?

Remove Google+ in install page.

links

google+mort

## Additional information
- PrestaShop version: 1.7.7.x
- PHP version: N/A

### PR DESCRIPTION
**Pull request by PululuK**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Remove Google+ in install page.  google+mort
| Type?         | improvement 
| Category?     | IN
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #20267.
| How to test?  | links

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20266)

### CODE REVIEW

**Comment by matks** [comment]
I will not click on "approve" button 😱 I am too afraid of Google.

Don't anger them! they will destroy our SEO 😨 !

**Comment by PululuK** [comment]
> I will not click on "approve" button  I am too afraid of Google.
> 
> Don't anger them! they will destroy our SEO  !

@google @droot @aamonten can you confirm please ? Thanks !

File: install-dev/theme/views/process.php [hunk 1]
```diff
@@ -113,9 +113,6 @@
 		<button type="button" class="btn-facebook" onclick="psinstall_facebook_click();">
 			<i></i> <?php echo $this->translator->trans('Share', array(), 'Install'); ?>
 		</button>
-		<button type="button" class="btn-google-plus" onclick="psinstall_google_click();">
```
Comment (Progi1984) [inline install-dev/theme/views/process.php:116]: Need to remove the function `psinstall_google_click` from JS !

Comment (PululuK) [reply install-dev/theme/views/process.php:116]: Sorry @Progi1984  I am getting very old ... Thanks !

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by PierreRambaud** [comment]
The pull request #17340 will be backported on the 1.7.7.x instead.
Thanks for your contribution

**Comment by PululuK** [comment]
> The pull request #17340 will be backported on the 1.7.7.x instead.
> Thanks for your contribution

Thanks @PierreRambaud  !

**Pull request closed without merging**